[2023-10-25T11:01:40.360+0530] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.check_accuracy scheduled__2022-11-21T00:00:00+00:00 [queued]>
[2023-10-25T11:01:40.365+0530] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.check_accuracy scheduled__2022-11-21T00:00:00+00:00 [queued]>
[2023-10-25T11:01:40.365+0530] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-25T11:01:40.373+0530] {taskinstance.py:1380} INFO - Executing <Task(BranchPythonOperator): check_accuracy> on 2022-11-21 00:00:00+00:00
[2023-10-25T11:01:40.377+0530] {standard_task_runner.py:57} INFO - Started process 31246 to run task
[2023-10-25T11:01:40.379+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'check_accuracy', 'scheduled__2022-11-21T00:00:00+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp0faauxiu']
[2023-10-25T11:01:40.380+0530] {standard_task_runner.py:85} INFO - Job 158: Subtask check_accuracy
[2023-10-25T11:01:40.401+0530] {task_command.py:415} INFO - Running <TaskInstance: first_dag.check_accuracy scheduled__2022-11-21T00:00:00+00:00 [running]> on host harika-Latitude-5511
[2023-10-25T11:01:40.440+0530] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='check_accuracy' AIRFLOW_CTX_EXECUTION_DATE='2022-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-11-21T00:00:00+00:00'
[2023-10-25T11:01:40.445+0530] {python.py:194} INFO - Done. Returned value was: inaccurate
[2023-10-25T11:01:40.445+0530] {python.py:227} INFO - Branch callable return inaccurate
[2023-10-25T11:01:40.445+0530] {skipmixin.py:173} INFO - Following branch inaccurate
[2023-10-25T11:01:40.448+0530] {skipmixin.py:239} INFO - Skipping tasks [('accurate', -1)]
[2023-10-25T11:01:40.466+0530] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=check_accuracy, execution_date=20221121T000000, start_date=20231025T053140, end_date=20231025T053140
[2023-10-25T11:01:40.510+0530] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-25T11:01:40.547+0530] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
